src/communication/network_interface_cl.c

```diff
@@ -8124,6 +8124,281 @@ logtb_dump_trantable (FILE * outfp)
 #endif /* !CS_MODE */
 }
 
+/*
+ * mmon_get_server_info -
+ *
+ * return: cubrid error
+ *
+ *   server_info(in/out): save memory usage information of the server
+ */
+int
+mmon_get_server_info (MMON_SERVER_INFO & server_info)
+{
+#if defined(CS_MODE)
+  char *buffer, *ptr, *temp_str;
+  int bufsize = 0;
+  OR_ALIGNED_BUF (OR_INT_SIZE + OR_INT_SIZE) a_reply;
+  char *reply = OR_ALIGNED_BUF_START (a_reply);
+  int req_error, dummy;
+  int error = NO_ERROR;
+
+  req_error =
+    net_client_request2 (NET_SERVER_MMON_GET_SERVER_INFO, NULL, 0, reply,
+			 OR_ALIGNED_BUF_SIZE (a_reply), NULL, 0, &buffer, &bufsize);
+  if (req_error)
+    {
+      assert (er_errid () != NO_ERROR);
+      error = er_errid ();
+    }
+  else
+    {
+      ptr = reply;
+      ptr = or_unpack_int (ptr, &dummy);
+      ptr = or_unpack_int (ptr, &error);
+    }
+
+  if (error == NO_ERROR)
+    {
```

Comment:
Nit. I understand it works from your elaboration, but how about changing `dummy` to `area_size` like other network interfaces and adding an assertion, that claims your analysis?
```suggestion
    {
       assert (area_size > 0);
```

Reply:
No, I don't think it's needed. Forget this.